ensime.el

```diff
@@ -49,6 +49,7 @@
 (require 'ensime-stacktrace)
 (require 'ensime-debug)
 (require 'ensime-editor)
+(require 'ensime-expand-region)
```

Comment:
I don't think that this is a hard dependency.  This change does not `(require 'expand-region)` anywhere. 

`ensime-expand-region.el` only `defvar`s a particular variable of Expand Region which is the main extension point, but doesn't use any other code from Expand Region.  And it only becomes active after Expand Region is loaded.